examples/player/index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="../build.js"></script>
+    <script src="../common/shaders/skyGradient.js"></script>
+
+<script>
+// any extras would go here
+
+
+AFRAME.registerComponent('auto-hand-controls', {
```

Comment:
a specific goal of the changes was to allow support for hand-controls (e.g. point to teleport) to abstract away which specific controllers are supported, as teleport really only needs the start and stop event to make that work